packages/flutter/test/widgets/gesture_detector_test.dart

```diff
@@ -439,6 +439,51 @@ void main() {
       await longPress(kLongPressTimeout + const Duration(seconds: 1)); // To make sure the time for long press has occurred
       expect(longPressUp, 1);
     }, variant: buttonVariant);
+
+    testWidgets('Competition', (WidgetTester tester) async {
```

Comment:
```suggestion
    testWidgets('Unfinished DoubleTap does not prevent competing Tap', (WidgetTester tester) async {
```
Or some better names you can think of.